build.gradle

```diff
@@ -23,6 +23,9 @@ repositories {
     maven {
         url 'https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/'
     }
+    maven {
+        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release-local/'
```

Comment:
can we move this into the `maven` block above?

Reply:
don't think so. I think each of these is a different configuration of a maven repository - can't configure multiple repositories at once.